### PR TITLE
libavfilter: Add Documentation to Queue

### PR DESCRIPTION
**Pull request by shubhanshu02**
## Description

1. Adds documentation to `queue.h` in `libavfilter/dnn`
2. Corrects spelling of the word `describe` in OpenVino backend

@guoyejun please review the changes.

### CODE REVIEW

File: libavfilter/dnn/queue.h [hunk 1]
```diff
@@ -22,20 +22,95 @@
...
+ * @brief Returns the length of the Queue
+ */
 size_t ff_queue_size(Queue *q);
 
+/**
+ * @brief Returns a pointer to the data
+ * at the head of the queue.
+ *
+ * @retval NULL if null pointer was passed
+ * or length of queue is empty
```
Comment (guoyejun) [inline libavfilter/dnn/queue.h]: or queue is empty,  or length of queue is 0

File: libavfilter/dnn/queue.h [hunk 2]
```diff
@@ -22,20 +22,95 @@
...
+ */
 void *ff_queue_peek_front(Queue *q);
+
+/**
+ * @brief Returns a pointer to the data at
+ * the tail of the queue.
+ *
+ * @retval NULL if null pointer was passed
+ * or length of queue is empty
```
Comment (guoyejun) [inline libavfilter/dnn/queue.h]: save as above

File: libavfilter/dnn/dnn_backend_openvino.c [hunk 3]
```diff
@@ -318,7 +318,7 @@ static DNNReturnType init_model_ov(OVModel *ov_model, const char *input_name, co
     }
 
     // all models in openvino open model zoo use BGR with range [0.0f, 255.0f] as input,
-    // we don't have a AVPixelFormat to descibe it, so we'll use AV_PIX_FMT_BGR24 and
+    // we don't have a AVPixelFormat to describe it, so we'll use AV_PIX_FMT_BGR24 and
```
Comment (guoyejun) [inline libavfilter/dnn/dnn_backend_openvino.c]: can be in a separate patch

File: libavfilter/dnn/queue.h [hunk 4]
```diff
@@ -22,20 +22,95 @@
 #ifndef AVFILTER_DNN_QUEUE_H
 #define AVFILTER_DNN_QUEUE_H
 
+/**
+ * Linear double-ended data structure
+ */
 typedef struct Queue Queue;
 
+/**
+ * @brief Creates a Queue instance.
```
Comment (guoyejun) [inline libavfilter/dnn/queue.h]: I did a quick check, and see 'do' instead of 'does'.   
Creates --> Create, and all others below.

Comment (shubhanshu02) [reply libavfilter/dnn/queue.h]: Okay, got it.

File: libavfilter/dnn/queue.h (hunk 1 above)
Comment (shubhanshu02) [reply libavfilter/dnn/queue.h]: Okay

File: libavfilter/dnn/dnn_backend_openvino.c (hunk 3 above)
Comment (shubhanshu02) [reply libavfilter/dnn/dnn_backend_openvino.c]: Okay.

**Comment by guoyejun** [comment]
you can use 'git push ... -f' to overwrite the branch, and no 'revert' is needed.

**Comment by shubhanshu02** [comment]
Okay, done!

**Comment by shubhanshu02** [comment]
Thank you for reviewing.
